### ISSUE TITLE
Testing YCSB configurations with compression

### ISSUE THREAD

**Issue by agorrod** [wiredtiger/wiredtiger#1768]
WiredTiger supports several different compression libraries, it would be good to have a performance comparison between them using an established workload.

An obvious candidate for a workload is the YCSB configurations we are testing in Jenkins (http://build.wiredtiger.com:8080/job/mongodb-perf-ycsb-develop/plot/).

It would be interesting to get numbers comparing no compression, zlib, bzip2, snappy and lz4 (once it is merged into the main code line). The statistics worth knowing are:
* Total throughput for different operation types
* Latency numbers
* On disk size

It would potentially be interesting to generate some content that can be added to the benchmarks section of the wiki page https://github.com/wiredtiger/wiredtiger/wiki#benchmarks.
